api/user/models.py

```diff
@@ -0,0 +1,23 @@
+from sqlalchemy import (Column, String, Integer, Enum)
+from flask_bcrypt import generate_password_hash
+
+from helpers.database import Base
+from utils.utility import StateType, Utility
+from utils.validators import validate_empty_fields
+
+
+class User(Base, Utility):
+    __tablename__ = 'users'
+    id = Column(Integer, primary_key=True)
+    email = Column(String, nullable=False)
+    name = Column(String, nullable=False)
+    state = Column(Enum(StateType), default='active')
+    password_hash = Column(String, nullable=False)
+
+    def __init__(self, **kwargs):
+        validate_empty_fields(**kwargs)
```

Comment:
I think this would have been better of done in the schema.py before the data reached the model

Reply:
Nice Catch. This has been worked on